src/engine/platform/opl.cpp

```diff
@@ -1666,10 +1668,7 @@ void DivPlatformOPL::renderSamples() {
   size_t memPos=0;
   for (int i=0; i<parent->song.sampleLen; i++) {
     DivSample* s=parent->song.sample[i];
-    int paddedLen=(s->lengthB+255)&(~0xff);
-    if ((memPos&0xf00000)!=((memPos+paddedLen)&0xf00000)) {
-      memPos=(memPos+0xfffff)&0xf00000;
-    }
+    int paddedLen=(s->lengthB+0x3)&(~0x3);
```

Comment:
Here

Reply:
Ah, right, I didn’t think about that! For the same reason I should omit no. 4 then. I dropped the last two commits.

I should’ve remembered the `s->offB` stuff since I previously dealt with it [for the OPL4 & MultiPCM](https://github.com/grauw/furnace/commit/6773249d1f98694e76295b9c3810dbe5ddc9aef8).